src/components/Dropdown/Dropdown.tsx

```diff
@@ -222,48 +229,93 @@ export default class Dropdown extends AutoControlledComponent<
             getRootProps,
             getToggleButtonProps,
             isOpen,
+            toggleMenu,
             highlightedIndex,
             selectItemAtIndex,
           }) => {
+            const accessibilityRootProps = getRootProps(
+              { refKey: 'innerRef' },
+              { suppressRefError: true },
+            )
+            const { innerRef, ...accessibilityRootPropsRest } = accessibilityRootProps
             return (
-              <div
-                className={classes.containerDiv}
-                onClick={this.handleContainerClick.bind(this, isOpen)}
-              >
-                {multiple && this.renderSelectedItems(styles)}
-                {search &&
-                  this.renderSearchInput(
-                    getRootProps,
-                    getInputProps,
+              <Ref innerRef={innerRef}>
```

Comment:
will leave the dropdown root on this div by having it in this ref, and will get the rest of the accessibility props and put them on the searchInput (that's the combobox). This way Downshift will work as expected and also the combobox props (that Downshift provides by the same getRootProps) will be applied to the searchInput.